frontend/mina-signer/src/MinaSigner.ts

```diff
@@ -340,6 +341,9 @@ class Client {
    */
   public signParty(party: Party, privateKey: PrivateKey): Signed<Party> {
     const parties = JSON.stringify(party.parties.otherParties);
+    if (party.feePayer.fee === undefined || party.feePayer.fee <= 0) {
+      throw "Fee must be greater than 0";
```

Comment:
should this also throw an error if `fee < getPartyMinimumFee(party.parties.otherParties)`?

Reply:
Yup, that is a better check for that condition, will update!